__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -481,6 +512,7 @@ describe('TESTING ChatSchema', () => {
         'userId',
         'channelId',
         'message',
+        [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 481, if the function expects a string (specifically a JSON string), then this should be changed.
```diff
-  [],
+  '',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->